hw/ip/hmac/data/hmac.hjson

```diff
@@ -4,6 +4,14 @@
 {
   name:               "hmac",
   one_line_desc:      "Accelerator for SHA-256-based keyed hash message authentication code and the SHA-256 hash function",
+  one_paragraph_desc: '''
+  The OpenTitan HMAC hardware block is a keyed hash based message authentication code generator using [SHA-256][nist-fips-180-4] to check the integrity of an incoming message and optionally a signature signed with the same secret key.
+  This HMAC implementation is not hardened against side-channel analysis (SCA) or fault injection (FI) attacks.
+  It is meant purely for hashing acceleration.
```

Comment:
I don't think we need this line. IMO it is covered by the sentence above.

Reply:
Hmm, I think it's valuable to have the security vs performance juxtaposition clearly in one sentence. We could merge the currently two sentences, though?

> This HMAC implementation is not hardened against side-channel analysis (SCA) or fault injection (FI) attacks; it is meant purely for hashing acceleration.

Reply:
Your suggestion sounds good to me @andreaskurth .